src/util/thread.h

```diff
@@ -26,6 +26,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "../threading/mutex_auto_lock.h"
 #include "porting.h"
 #include "log.h"
+#include "container.h" // MutexedQueue
```

Comment:
That just means that even the order of inclusion is important; I need to include "util/container.h" in mesh_generator_thread.h *before* "util/thread.h", otherwise there's a compiler error.